scrunch/tests/test_datasets.py

```diff
@@ -1218,10 +1218,26 @@ def test_edit_alias(self):
         ds.resource.body['streaming'] = 'no'
         var = ds['var1_alias']
         var.edit(alias='test1')
-        # Reading another variable breaks because `alias` has been removed(0
-        # from _IMMUTABLE_ATTRIBUTES already. Use .discard()
         var2 = ds['var2_alias']
 
+    def test_edit_resolution(self):
+        variables = {
+            '001': {
+                'id': '001',
+                'alias': 'datetime_var',
+                'name': 'Datetime Variable',
+                'type': 'datetime',
+                'is_subvar': False,
+                'view': {'rollup_resolution': 'ms'}
```

Comment:
In order to make the test verify that it's not overwriting other attributes you need to have this initial `view` value contain some other key. And then assert that both keys are indeed still there.